chase/frontend/components/cookie_banner.tsx

```diff
@@ -19,11 +20,22 @@ export default function CookieBanner() {
 
   const handleAcceptButtonClick = () => {
     cookieConsentRef?.current?.accept();
+    setIsVisible(false);
   };
 
-  // TODO: Fix cookie interference with the backend
+  useEffect(() => {
+    if (cookieConsentRef.current) {
+      setIsVisible(cookieConsentRef.current.state.visible);
+    }
+  }, [cookieConsentRef.current?.state.visible]);
+
+
+   // TODO: Fix cookie interference with the backend
```

Comment:
@m1212e is this even a problem anymore?

Reply:
Probably not. If I'm done with the delegator auth flow, I'll try to adopt the same in CHASE. That'll fix this I think.